drivers/hmis/app/models/hmis/form/definition.rb

```diff
@@ -204,13 +231,14 @@ class Hmis::Form::Definition < ::GrdaWarehouseBase
   end
 
   def self.find_definition_for_role(role, project: nil, version: nil)
```

Comment:
I don't think this function needs to accept `version` at all honestly. I don't know why I added that. The point of version is that the most recent one is the active one. If you wanted to look up a specific older definition  you would do that by the definition ID, not this.

What you have here is fine, but I think deleting this `version` arg and removing that test case would be just as good. Sorry about that